js/index.js

```diff
@@ -2,8 +2,151 @@ const modal = document.querySelector('#menuModal');
+    img: 'images/projects/gps_project.jpg',
+    technologies: ['C++', 'At328', 'At2560', 'Sim808'],
+    linkToLive: 'https://github.com/LuisDavidRodriguez',
+    linkToSource: 'https://bitbucket.org/david5048/gpsarduino-2/src/master/',
+  },
+  {
+    name: 'Automovile Acelerometer',
+    description: 'Accelerometer sensor and flow sensor. This is another software and hardware related to the GPS system, I add this feature because the drivers has a totally careless driving way, so the vehicles were being regularly struck with rocks or another obstacles resulting in staggering maintenance bills for the company. Now each car has a Accelerometer sensor, which is read by a microcontroler each 10ms, when the G-forces goes out from certain levels the software convert the analog lecture to a digital value and categorize the lecture according to one given scale',
+    img: 'images/projects/acelerometer.png',
+    technologies: ['C++', 'PHP', 'SQL', 'Eagle'],
+    linkToLive: 'https://github.com/LuisDavidRodriguez',
+    linkToSource: 'https://bitbucket.org/david5048/acelerometrodefinitivo/src/master/',
+  }];
+
+function populateModal(number, arr) {
+  const modalProjectTitle = modalProject.querySelector('h3');
+  const modalProjectList = modalProject.querySelector('ul');
+  const modalProjectImage = modalProject.querySelector('img');
+  const modalProjectDescription = modalProject.querySelector('p');
+  const modalProjectSeeLiveBtn = modalProject.querySelector('#seeLiveBtn');
+  const modalProjectSeeSourceBtn = modalProject.querySelector('#seeSourceBtn');
+
+  const project = arr[number];
+  modalProjectTitle.innerText = project.name;
+  modalProjectDescription.innerText = project.description;
+  modalProjectImage.src = project.img;
+  modalProjectSeeLiveBtn.setAttribute('href', project.linkToLive);
+  modalProjectSeeSourceBtn.attributes.href.nodeValue = project.linkToSource;
+
+  // add dinamically the li elements into the list
+  modalProjectList.innerHTML = '';
+  project.technologies.forEach((item) => {
+    const element = document.createElement('li');
+    element.innerText = item;
+    element.classList.add('sto');
+    modalProjectList.append(element);
+  });
+}
```

Comment:
- [ ] Kindly stick as much as possible with Figma design color, font, and text. Your heading has different font and color and `li` tag has a different color and different border kindly fix it
### your design
![your](https://user-images.githubusercontent.com/81727988/179897959-095edc96-6999-43b5-90b0-696a4be2b7df.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/179898014-6f8f6d92-8084-488b-8e23-40ac1eaaa7a7.png)

Reply:
Solved 😄